patches/tModLoader/Terraria.ModLoader.Engine/Steam.cs

```diff
@@ -49,5 +51,14 @@ public static bool EnsureSteamAppIdTerrariaFile()
 			File.WriteAllText("steam_appid.txt", TerrariaAppID.ToString());
 			return exists;
 		}
+
+		public static bool CheckSteamCloudStorageSufficient()
+		{
+			if (IsSteamApp)
```

Comment:
if (IsSteamApp && other && other)

Reply:
should I still do line breaks after each &&?